packages/vue-cli-plugin-vuetify/generator/templates/v3/src/plugins/vuetify.js

```diff
@@ -1,8 +1,8 @@
 import '@mdi/font/css/materialdesignicons.css'
```

Comment:
I think we should remove the direct mdi icon import and include a link tag in the index.html file.

Reply:
CDNs suck

Reply:
I'm not a fan of using CDNs, but if this is a change that's wanted I will make it.

Reply:
Yes, please move to a cdn. For presets/plugins that use webfontloader, it would be put there.

Reply:
That's done in another PR, so I'm going to leave it out of this one.